karamel-ui/deploy-packages-server.sh

```diff
@@ -20,9 +20,11 @@ exec "$JAVACMD" $JAVA_OPTS -Xms128m -Xmx4g \
   -Dapp.pid="$$" \
   -Dapp.repo="$REPO" \
   -Dapp.home="$BASEDIR" \
-  -Dhttp.proxyHost="$http_proxy"\
+  -Dhttp.proxy="$http_proxy"\
+  -Dhttp.proxyHost="$http_proxy_host"\
   -Dhttp.proxyPort="$http_proxy_port"\
-  -Dhttps.proxyHost="$https_proxy"\
+  -Dhttps.proxy="$https_proxy"\
+  -Dhttps.proxyHost="$https_proxy_host"\
```

Comment:
No, this is wrong. I already fixed it once here: https://github.com/karamelchef/karamel/commit/ddeeb336bafc20d08ff30b805dfc19ca553e6c73

See the documentation:
https://docs.oracle.com/javase/8/docs/technotes/guides/net/proxies.html

Reply:
I get them now as system property properties, not env variables. 
I think that is all your comments addressed.

Reply:
An update. This won't work for HTTP proxies. I have not managed to get 'git clone ...' to work with a http_proxy, only a https_proxy. I updated the docs in hopsworks-installer.sh to reflect this.

Reply:
To be clearer, in hopsworks-installer.sh, when i start karamel, i set the env variables http_proxy and https_proxy.
However, in the code above I also set the system properties when starting karamel: https.proxy and http.proxy to be the same as the env variables.